codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/Instantiator.kt

```diff
@@ -448,12 +450,10 @@ open class Instantiator(
 
     /**
      * Returns a default value for a shape.
-     *
-     * Warning: this method does not take into account any constraint traits attached to the shape.
```

Comment:
This is still accurate: `Instantiator` doesn't instantiate with values that honor constraint traits.